### PR TITLE
fix error in Fortran_to_Aviary.py

### PR DESCRIPTION
**Pull request by xjjiang**
### Summary

If users run `fortran_to_aviary` without specifying an output file, it runs into error.
If GASP data file has `ARNGE(1) = 0.`, it runs into error.

This PR should fix both.

### Related Issues

- Resolves #

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: aviary/utils/Fortran_to_Aviary.py [hunk 1]
```diff
@@ -362,6 +362,7 @@ def update_gasp_options(vehicle_data):
         # if the design range target_range value is 0, set the problem_type to fallout
         if design_range[0] == 0:
             input_values.set_val('problem_type', ['fallout'])
+            problem_type = 'fallout'
```
Comment (crecine) [inline aviary/utils/Fortran_to_Aviary.py:365]:
```suggestion
            problem_type = 'fallout'
            input_values.set_val('problem_type', [problem_type ])
```

Comment (crecine) [reply aviary/utils/Fortran_to_Aviary.py:365]: this way the string `'fallout'` only has to be entered/updated once if things change

File: aviary/utils/Fortran_to_Aviary.py [hunk 2]
```diff
@@ -613,5 +614,8 @@ def _exec_F2A(args, user_args):
         args.input_deck = args.input_deck[0]
     filepath = args.input_deck
 
-    create_aviary_deck(filepath, args.legacy_code, args.defaults_deck,
-                       Path(args.out_file), args.force)
+    if args.out_file:
+        create_aviary_deck(filepath, args.legacy_code, args.defaults_deck,
+                           Path(args.out_file), args.force)
+    else:
+        create_aviary_deck(filepath, args.legacy_code, args.defaults_deck, args.force)
```
Comment (crecine) [inline aviary/utils/Fortran_to_Aviary.py:621]:
```suggestion
      create_aviary_deck(filepath, args.legacy_code, args.defaults_deck,
                           args.out_file, args.force)
```

File: aviary/utils/Fortran_to_Aviary.py [hunk 3]
```diff
@@ -57,7 +57,7 @@ def create_aviary_deck(fortran_deck: str, legacy_code=None, defaults_deck=None,
 
     if not out_file:
         name = fortran_deck.stem
```
Comment (crecine) [inline aviary/utils/Fortran_to_Aviary.py:59]:
```suggestion
    if out_file:
        out_file = Path(out_file)
    else:
        name = fortran_deck.stem
```

**Review by crecine — COMMENTED** [review]
Thanks for finding this, I made a couple suggestions

File: aviary/utils/Fortran_to_Aviary.py (hunk 1 above)
Comment (xjjiang) [reply aviary/utils/Fortran_to_Aviary.py:365]: agree. done

File: aviary/utils/Fortran_to_Aviary.py (hunk 3 above)
Comment (xjjiang) [reply aviary/utils/Fortran_to_Aviary.py]: done.

**Review by crecine — APPROVED** [review]

**Review by johnjasa — APPROVED** [review]

**Review by jkirk5 — APPROVED** [review]
Looks good

**Pull request merged**